src/view/app/Query/Update/update.tsx

```diff
@@ -178,21 +192,55 @@ export default function UpdatePopup({
                     </div>
                 )}
             </Popup>
-            <ProBroButton
-                startIcon={<AddIcon />}
-                onClick={selectedRows.size === 1 ? copyRecord : insertRecord}
-                disabled={selectedRows.size > 0 ? false : false}
-            >{selectedRows.size === 1 ? "Copy" : "Create"}</ProBroButton>
-            <ProBroButton
-                startIcon={<EditIcon />}
-                onClick={updateRecord}
-                disabled={selectedRows.size === 1 ? false : true}
-            >Update</ProBroButton>
-            <ProBroButton
-                startIcon={<DeleteIcon />}
-                onClick={deleteRecord}
-                disabled={selectedRows.size > 0 ? false : true}
-            >Delete</ProBroButton>
+
+
+
+            {isWindowSmall ? (
+                <>
+                    <ProBroButton
+                        startIcon={<AddIcon />}
+                        onClick={selectedRows.size === 1 ? copyRecord : insertRecord}
+                        disabled={selectedRows.size > 0 ? false : false}
+                    >
+                    </ProBroButton>
+                    <ProBroButton
+                        startIcon={<EditIcon />}
+                        onClick={updateRecord}
+                        disabled={selectedRows.size === 1 ? false : true}
+                    >
+                    </ProBroButton>
+                    <ProBroButton
+                        startIcon={<DeleteIcon />}
+                        onClick={deleteRecord}
+                        disabled={selectedRows.size > 0 ? false : true}
+                    >
+                    </ProBroButton>
+                </>
+            ) : (
+                <>
```

Comment:
pretty good, but could you make buttons position in there original place and move them down just when they don't fit 
![image](https://github.com/BalticAmadeus/ProBro/assets/78811378/0cc32405-77bb-46bb-9c30-a225e151d22c)